src/pulselistener/tests/test_mercurial.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
```

Comment:
It'd be nice to also test the case where a stack of revisions applies on top of a base revision which is not the latest in central, but we can also add this test later on.

Reply:
I added a unit test for this case